### PR TITLE
#167 - Accessing layers from HTTP slices via Repo.layers

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #167 
Reading and writing layers using `Repo.layers()` in HTTP slices instead of `BlobStore`

### CODE REVIEW

**Comment by olegmoz** [comment]
@g4s8 please review or assign a reviewer for this PR

**Comment by g4s8** [comment]
@olegmoz you can request review from `artipie/contributors` alias, you'll be able to merge if any team member accept the PR.

File: src/main/java/com/artipie/docker/http/BlobEntity.java [hunk 1]
```diff
@@ -194,4 +181,58 @@ public Response response(
...
+        /**
+         * Matches request path by RegEx pattern.
+         *
+         * @return Path matcher.
+         */
+        private Matcher path() {
+            final String path = new RequestLineFrom(this.line).uri().getPath();
+            final Matcher matcher = PATH.matcher(path);
+            if (!matcher.matches()) {
+                throw new IllegalStateException(String.format("Unexpected path: %s", path));
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/docker/http/BlobEntity.java:233]: @olegmoz I've noticed that this branch is not tested in `BlobEntityGetTest`, let's add test method for it or corresponding todo/issue, please.

Comment (olegmoz) [reply src/main/java/com/artipie/docker/http/BlobEntity.java:235]: @olenagerasimova thanks, added a puzzle for that

**Comment by olenagerasimova** [comment]
@olegmoz please format the puzzle properly

**Comment by olegmoz** [comment]
@olenagerasimova fixed the puzzle, please approve

**Pull request merged**